client/src/components/createContestComponents/Alert.js

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect, useContext } from 'react';
+import MuiAlert from '@material-ui/lab/Alert';
+import { Snackbar } from '@material-ui/core';
+import { ContestContext } from '../../context/ContestContext';
+
+function Alert(props) {
```

Comment:
props are not used so it can just be: function Alert() {

Reply:
Yep!